thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -517,9 +516,10 @@ public String serializeExampleRequest(String serviceName, String methodName,
         }
 
         final TBase<?, ?> exampleTBase = (TBase<?, ?>) exampleRequest;
-        final TSerializer serializer = new TSerializer(ThriftProtocolFactories.TEXT);
         try {
-            return serializer.toString(exampleTBase, StandardCharsets.UTF_8.name());
+            final TSerializer serializer = new TSerializer(ThriftProtocolFactories.TEXT);
+            // TSerializer.toString(TBase, charset) was removed in Thrift 0.14.0
+            return serializer.toString(exampleTBase);
```

Comment:
We have to use `toString(TBase, String)` if it exists, because an old version of `toString(TBase)` uses the system default charset, which may not be UTF-8.

Reply:
That's a good idea.